### ISSUE TITLE
yoast premium

### ISSUE THREAD

**Issue by mnights101** [mnights101/drnights#3]
The yoast plugin premium seems to be having an issue, it is not displaying  correctly in admin area, maybe upgrading it throught wp-admin may resolve this. I also see there is a new theme update.

![yoast-issue](https://user-images.githubusercontent.com/10249755/41071301-f5efa64c-69c4-11e8-807f-046600ac163b.gif)

**Comment by brogrammersagency** [comment]
Yes I have updates yoast seo plugin from admin but still giving update message... Can you again check Yoast seo premium plugin site to get latest files zip. May be we have old version.

And also The7 new theme available for update if you can provide that I can also update the the7 theme.

**Comment by mnights101** [comment]
here is latest theme The7 
[themeforest-5556590-the7-responsive-multipurpose-wordpress-theme-wordpress-theme.zip](https://github.com/mnights101/drnights/files/2091617/themeforest-5556590-the7-responsive-multipurpose-wordpress-theme-wordpress-theme.zip)

here the latetest yoast:
[wordpress-seo-premium-7.6.1.zip](https://github.com/mnights101/drnights/files/2091768/wordpress-seo-premium-7.6.1.zip)
